sdk/ml/azure-ai-ml/tests/automl_job/e2etests/test_automl_image_classification.py

```diff
@@ -83,6 +83,8 @@ def test_image_classification_multiclass_run(
 
         training_data = Input(type=AssetTypes.MLTABLE, path=train_path)
         validation_data = Input(type=AssetTypes.MLTABLE, path=val_path)
+        properties = get_automl_job_properties()
+        properties['_pipeline_id_override'] = "azureml://registries/azmlft-dev-registry01/components/image_classification_pipeline/versions/0.0.5"
```

Comment:
any way we can use the latest version instead of a specific one? also, why do we need to specify it?

Reply:
We are specifying the pipeline component id because JOS currently points to azureml-dev registry which is not accessible outside of centraluseuap. However, the nighly sdk tests run in eastus, so we need to have components in a registry that is accessible in prod regions.

Reply:
Will check on latest version, it should be possible